controls/SV-204392.rb

```diff
@@ -3,23 +3,26 @@
     and group membership of system files and commands match the vendor values.'
   desc 'Discretionary access control is weakened if a user or group has access permissions to system files and
     directories greater than the default.'
-  desc 'check', %q(Verify the file permissions, ownership, and group membership of system files and commands match the
-    vendor values.
-    Check the default file permissions, ownership, and group membership of system files and commands with the following
-    command:
-    # for i in `rpm -Va | egrep '^.{1}M|^.{5}U|^.{6}G' | cut -d " " -f 4,5`;do for j in `rpm -qf $i`;do rpm -ql $j
```

Comment:
`egrep` is the same as `grep -E`. All other changes are semantic. This control does not need updated logic.